### ISSUE TITLE
Inkscape extension does not work in Ubuntu 20.04

### ISSUE THREAD

**Issue by bianchifan** [inkcut/inkcut#247]
System: Ubuntu 20.04 upgraded from 19.10
Inkscape 1.0 for 0.92.5 didn't ran any extension

Inkcut is running fine standalone but you can't call it from inkscape, neither marked area, nor complete file
![Bildschirmfoto von 2020-05-15 18-00-14](https://user-images.githubusercontent.com/17491743/82139752-79ef5d80-982a-11ea-83b6-85f462a56d6f.png)
![Bildschirmfoto von 2020-05-15 18-00-53](https://user-images.githubusercontent.com/17491743/82139759-84115c00-982a-11ea-9f52-715781b58e7a.png)

EDIT:
The py-scripts from the actual master were used (15.05.2020)

**Comment by aaujon** [comment]
I get the same result with python >= 3.6, it's related to subprocess : 
`Changed in version 3.6: Popen destructor now emits a ResourceWarning warning if the child process is still running.`
I will try to have a look this week.

**Comment by jisep32** [comment]
I'm getting a different error but would this be related?

/usr/lib/python3.8/subprocess.py:942: ResourceWarning: subprocess 9731 is still running
  _warn("subprocess %s is still running" % self.pid,
ResourceWarning: Enable tracemalloc to get the object allocation traceback

**Comment by aaujon** [comment]
Yes it's the same issue, inkscape is now using python 3 with ubuntu 20 and fedora 32

**Comment by bianchifan** [comment]
No resource warnings any more but it still does not run :(
After a few seconds a box flashes for less than 0.1 sec, so only 1 clear pic on a screen recording..
![Bildschirmfoto von 2020-05-25 17-00-37](https://user-images.githubusercontent.com/17491743/82824982-a2bac700-9eaa-11ea-99f6-23470ebfaa09.png)
"Open current document..." is working, please wait

But nothing happens

**Comment by aaujon** [comment]
Hmm it's working for me, I can have a look later this week

**Comment by jisep32** [comment]
I'm having the same warning flash for less then 0.1 seconds but Inkcut never open's.  Inkcut will run fine through terminal in Ubuntu 20.04.

**Comment by bianchifan** [comment]
> Hmm it's working for me, I can have a look later this week

I tried to catch what happens, maybe it can help.
Gnome System utilitie are too slow, so I tried with top
2 sceneries, cut selection and current document, both started with clean inkscape

 selection
- python call 
- three! more inkscape jobs
- python disappears, inkcut appears very, very short (hard to catch)
- 4 inkscape jobs do remain, 3 without noticable CPU usage
![Bildschirmfoto von 2020-05-26 13-04-24](https://user-images.githubusercontent.com/17491743/82999492-5afe8180-a009-11ea-8d58-9f94750145e0.jpg)

![Bildschirmfoto von 2020-05-26 11-45-26](https://user-images.githubusercontent.com/17491743/82999509-62be2600-a009-11ea-8494-25665f98f142.jpg)

current document
- python call with a temp document
- python disappeares, inkcut never visible for me due to delays when running SSR
- 2 more inkscape jobs
- 3 inkscape jobs will stay with two of them almost doing nothing
![Bildschirmfoto von 2020-05-26 12-58-05](https://user-images.githubusercontent.com/17491743/82999982-0f98a300-a00a-11ea-80ec-f851774dd4f7.jpg)

The fastest video I could catch with both scenes and top/SSR
[simplescreenrecorder-2020-05-26_13.09.31.zip](https://github.com/inkcut/inkcut/files/4687724/simplescreenrecorder-2020-05-26_13.09.31.zip)

**Comment by aaujon** [comment]
Hi,
Thanks for you report, can you check the inkcut log (probable under ~/.config/inkcut/logs/inkcut.log)?

**Comment by bianchifan** [comment]
Oh.., I forgot to mention, no log was produced, no file inkcut.txt  created.

**Comment by aaujon** [comment]
I was finally able to reproduce this issue (I hope), I have the same behaviour when the inkcut path is incorrect. There is now an error message with code 2 if the path is wrong.
I also replaced a deprecated call, can you try with https://github.com/inkcut/inkcut/pull/257 ?

**Comment by bianchifan** [comment]
I just tried..

Cut selection:
![Bildschirmfoto von 2020-06-09 13-52-29](https://user-images.githubusercontent.com/17491743/84144589-c9efc780-aa58-11ea-988d-bfffe5640849.png)

Current document:
![Bildschirmfoto von 2020-06-09 13-52-59](https://user-images.githubusercontent.com/17491743/84144610-d6742000-aa58-11ea-8c31-6e21ad94a275.png)

**Comment by aaujon** [comment]
I just fixed the bad indentation, bad copy/paste .. sorry
For the 'Error while starting inkcut', how did you install inkcut ? if it's not using pip, 'inkcut' binary is probably not in the path (if you use an alias for inkcut it doesn't work) can you try using DEBUG = True and set correct value line 58 and 59 ?

**Comment by bianchifan** [comment]
excuse me please but I don't see any difference between old line 72 and new...
inkcut should be in the path..
![Bildschirmfoto von 2020-06-09 21-36-49](https://user-images.githubusercontent.com/17491743/84191810-5cb05680-aa99-11ea-95fb-246b25ed91ac.png)
Originally it was installed with pip/pypi??
After the Ubuntu upgrade I only copied the changed extension scripts, inkscape itself is still just the same.

**Comment by bianchifan** [comment]
Sorry;; I'm handicaped at the moment, I pressed the wrong button :((

**Comment by aaujon** [comment]
Can you try the command : whereis inkcut ?

**Comment by bianchifan** [comment]
![Bildschirmfoto von 2020-07-07 17-50-59](https://user-images.githubusercontent.com/17491743/86807923-9f565600-c07a-11ea-920f-1dd16b5ab7f0.png)
![Bildschirmfoto von 2020-07-07 17-51-05](https://user-images.githubusercontent.com/17491743/86807919-9d8c9280-c07a-11ea-92c6-1d1d771d441c.png)
